amplience-automation/automation-files/schema/schemas/all-definitions-schema.json

```
@@ -397,6 +400,10 @@
                                         "toolName": "clear_formatting"
                                     }
                                 ]
+                            },
+                            {
+                              "type": "button",
```

Spacing (4-space tabs)
```suggestion
                                "type": "button",
```